tutorials/Desktop_Components/README.md

```diff
@@ -254,7 +271,6 @@ The main arguments of the `BoxWithTooltip` function:
  - offset - tooltip offset, the default position of the tooltip is under the mouse cursor
```

Comment:
We now have `tooltipPlacement` insted of `offset`

Reply:
Done